TelegramBot.py

```diff
@@ -1,13 +1,13 @@
 # Server Monitoring Bot
-import socket
-import json
 import argparse
+import json
+import socket
 import logging
-import time
 import datetime
-import requests
+import time
 from enum import Enum
 
+import requests
 
```

Comment:
By adding this import statement the 2 blank lines required between import statements and function are reduced to 1 blank line. Hence, this introduces error E302 report by the ci [here](https://travis-ci.org/Feathercoin-Foundation/Server-Monitoring-Bot/jobs/428988887#L473). Alternatively, this can also be observed by running `flake8` locally.